variables.md

```diff
@@ -113,6 +118,62 @@ PERF_KERNEL | boolean | false | Enables kernel performance testing.
 PERF_INSTALL | boolean | false | Enables kernel performance testing installation part.
 PERF_SETUP | boolean | false | Enables kernel performance testing deployment part.
 PERF_RUNCASE | boolean | false | Enables kernel performance testing run case part.
+_SECRET_PUBLIC_CLOUD_REST_URL | string | "https://publiccloud.qa.suse.de/vault" | Vault server URL
+_SECRET_PUBLIC_CLOUD_REST_USER | string | "" | Vault server user name
+_SECRET_PUBLIC_CLOUD_REST_PW | string | "" | Vault server user password
+_SECRET_PUBLIC_CLOUD_REST_SSL_INSECURE | boolean | true | Do not validate Vault server certificate
+PUBLIC_CLOUD | boolean | false | All Public Cloud tests have this variable set to true. Contact: qa-c@suse.de
+PUBLIC_CLOUD_ACCOUNT | string | "" | 
+PUBLIC_CLOUD_ACCNET | boolean | false | If set, az_accelerated_net test module is added to the job.
+PUBLIC_CLOUD_ARCH | string | "x86_64" | The architecture of created VM.
+PUBLIC_CLOUD_AZURE_OFFER | string | "" | 
+PUBLIC_CLOUD_AZURE_SKU | string | "" | 
+PUBLIC_CLOUD_BUILD | string | "" | The image build number. Used only when we use custom built image.
+PUBLIC_CLOUD_BUILD_KIWI | string | "" | The image kiwi build number. Used only when we use custom built image.
+PUBLIC_CLOUD_CHECK_BOOT_TIME | boolean | false | If set, boottime test module is added to the job.
+PUBLIC_CLOUD_CLIENT_ID | string | "" | 
+PUBLIC_CLOUD_CONFIDENTIAL_VM | boolean | false | GCE Confidential VM instance
+PUBLIC_CLOUD_CONSOLE_TESTS | boolean | false | If set, console tests are added to the job.
+PUBLIC_CLOUD_CONTAINERS | boolean | false | If set, containers tests are added to the job.
+PUBLIC_CLOUD_EC2_UPLOAD_AMI | string | "" | 
+PUBLIC_CLOUD_EC2_UPLOAD_SECGROUP | string | "" | 
+PUBLIC_CLOUD_EC2_UPLOAD_VPCSUBNET | string | "" | 
```

Comment:
```suggestion
PUBLIC_CLOUD_EC2_UPLOAD_VPCSUBNET | string | "" | Allow to instruct ec2uploadimg script to use some existing VPC instead of creating new one.
```